pkg/asset/internal/templates.go

```diff
@@ -1079,13 +1079,38 @@ spec:
         - name: flannel-cfg
           mountPath: /etc/kube-flannel/
       - name: install-cni
-        image: {{ .Images.Busybox }}
-        command: [ "/bin/sh", "-c", "set -e -x; TMP=/etc/cni/net.d/.tmp-flannel-cfg; cp /etc/kube-flannel/cni-conf.json ${TMP}; mv ${TMP} /etc/cni/net.d/10-flannel.conf; while :; do sleep 3600; done" ]
+        image: {{ .Images.Alpine }}
+        command: 
```

Comment:
This scares me. I don't think we should be wget-ing the cni release at runtime. My initial reaction would be that we need a versioned container image that has all the files we need, and we're simply copying them to a different mount.